packages/storybook-addon/package.json

```diff
@@ -48,7 +48,7 @@
     "clean": "rimraf dist",
     "check": "tsc --noEmit",
     "lint": "eslint --ext=js,jsx,ts,tsx .",
-    "build": "rollup -c ../../scripts/rollup/config.js"
+    "build": "rollup -c ../../scripts/rollup/config.js && cp src/global.d.ts dist/global.d.ts"
```

Comment:
If you add the prepare script it'll check whether the structure of `package.json` works and make sure that `/preact` has a non-ESM entrypoint